app/infrastructure/firebase/_rest_client.py

```
@@ -46,23 +48,22 @@ async def _request_async(
-        if method == "DELETE":
-            return {}
-        raw = resp.content
-        return json.loads(raw.decode()) if raw else {}
+    if method == "GET":
+        resp = await client.get(url, headers=headers)
+    elif method == "PATCH":
+        resp = await client.patch(url, headers=headers, json=body)
+    elif method == "DELETE":
+        resp = await client.delete(url, headers=headers)
+    else:
+        raise ValueError(f"Unsupported method: {method!r}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)